src/deepmol/compound_featurization/mixed_descriptors.py

```diff
@@ -44,7 +47,8 @@ def _featurize(self, mol: Mol):
                 final_features = np.concatenate((final_features, current_features))
 
         except Exception:
-            print('error in smile: ' + str(mol))
+            self.logger = Logger()
```

Comment:
Yes, we need it because the class of the featurizer needs to be pickled. I had to "fool" the pickle function and transform the self.logger instance variables into strings. Because of that, we have to instantiate it again to access the class's global instance and true variables. It is a bit confusing, but it is a workaround for the fact that the python logging package is not pickable.

Reply:
Oh yeah, you already explained that! I forgot about it!